packages/infra/client/isolate-v8-runner/src/isolate.rs

```diff
@@ -146,8 +146,8 @@ pub async fn run_inner(
 
 	tracing::info!(?actor_id, "isolate kv initialized");
 
-	// Should match the path from `Actor::download_image` in manager/src/actor/setup.rs
-	let entrypoint = actor_path.join("js-bundle").join("index.js");
+	// Should match the path from `Actor::make_fs` in manager/src/actor/setup.rs
+	let entrypoint = actor_path.join("fs").join("index.js");
```

Comment:
similar to above, imo we should have a "source dir" or something for the actor. thoughts?

Reply:
This is the entire file system for the isolate though, do you want it to be fs/src?

Reply:
see above